vignettes/ORFikOverview.Rmd

```diff
@@ -0,0 +1,83 @@
+---
+title: "ORFik Overview"
+author: "Håkon Tjeldnes"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{ORFik Overview}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Welcome to the `ORFik` package. This vignette will walk you through our main 
+package usage with two examples, one easy and one real world example.  You will learn how to interpret 
+results, make features, and plot results.
+This package, `ORFik`, is created for fast and analysis of Open Reading frames, and to make less people reinvent the wheel.
+
+# Introduction
+
+`ORFik` main usage is to get Open reading frames from a fasta file and a gtf file describing the sequences, what you do with them, make features, plots etc, is up to you.
+
+# Easy example
+
+```r
+library(ORFik)
+
+# Make easy example data, here we dont use fasta file, but just a character vector:
```

Comment:
1. dont call anything easy example, for some people it might be still hard and you automatically condescend on them, say: Character vector example.
2.